CHANGELOG.md

```diff
@@ -25,12 +25,12 @@ These changes likely break some apps, please read the
 * The second parameter of `theme.NewThemedResource` was removed, it was previously ignored
 * The desktop.Cursor definition was renamed desktop.StandardCursor to make way for custom cursors
 
+* iOS apps preferences will be lost in this upgrade as we move to more advanced storage
```

Comment:
Is this relevant to this PR?

Reply:
I was updating the changelog and realised it would get too long if we listed every change, so removed the methods to leave just actual type removal.
So in a way it's related, yes.

Reply:
I have extracted the changed not strictly caused by this PR. Should be cleaner now.